openshift/resources.yaml

```diff
@@ -0,0 +1 @@
+../resources.yaml
```

Comment:
i'm a little confused by this file

Reply:
It's a symbolic link.  Let me verify that it downloads correctly (it's supposed to, I read about it :) )

I put it here so that we could get rid of the instruction on the Get Started page but still have coverage for stuff older than "latest", but using the current resources.yaml.

Reply:
@elmiko, yeah, just checked out a fresh repo and pulled this PR, the symbolic link is recreated in openshift dir

Reply:
thanks! i hadn't seen a link on the github side before